cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -53,7 +53,7 @@ object CqlParser extends JavaTokenParsers
    */
   def constant: Parser[Constant] = {
     import Constants._
-    (string | uuid | number | boolean) ^^ Constant // | hex // TODO
+    (string | uuid | number | boolean) ^^ StringConstant // | hex // TODO
```

Comment:
Now we have a separate class for every type of constant, shouldn't we start using them instead of mapping everything as a string?